src/main/java/com/xpandit/plugins/xrayjenkins/Utils/BuilderUtils.java

```diff
@@ -28,4 +31,12 @@ public static boolean isSupportedJobType(Class<? extends AbstractProject> jobTyp
                 || MavenModuleSet.class.isAssignableFrom(jobType);
     }
 
+    /**
+     * Utility method to check if the test execution key is invalid
+     * @param testExecKey the test execution key
+     * @return <code>true</code> if the test execution key matches the regex or is null/empty, <code>false</code> otherwise
+     */
+    public static boolean isTestExecKeyInvalid(String testExecKey) {
```

Comment:
This name could be something more generic.
Notice that you are checking if a given value it's or not a Env. Variable.